flake.lock: Update

Artturin opened the pull request:
Flake lock file updates:

• Updated input 'hydra':
    'github:NixOS/hydra/f48f00ee6d5727ae3e488cbf9ce157460853fea8' (2022-12-23)
  → 'github:NixOS/hydra/082495e34e094cae1eb49dbfc5648938e23c6355' (2023-03-27)
• Updated input 'hydra/nix':
    'github:NixOS/nix/11e45768b34fdafdcf019ddbd337afa16127ff0f' (2022-08-27)
  → 'github:nixos/nix/4acc684ef7b3117c6d6ac12837398a0008a53d85' (2023-02-22)
• Updated input 'hydra/nix/nixpkgs':
    'github:NixOS/nixpkgs/365e1b3a859281cf11b94f87231adeabbdd878a2' (2022-07-13)
  → 'github:NixOS/nixpkgs/04a75b2eecc0acf6239acf9dd04485ff8d14f425' (2022-12-08)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/6c8644fc37b6e141cbfa6c7dc8d98846c4ff0c2e' (2023-01-11)
  → 'github:NixOS/nixpkgs/555daa9d339b3df75e58ee558a4fec98ea92521e' (2023-04-18)

Artturin committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'hydra':
    'github:NixOS/hydra/f48f00ee6d5727ae3e488cbf9ce157460853fea8' (2022-12-23)
  → 'github:NixOS/hydra/082495e34e094cae1eb49dbfc5648938e23c6355' (2023-03-27)
• Updated input 'hydra/nix':
    'github:NixOS/nix/11e45768b34fdafdcf019ddbd337afa16127ff0f' (2022-08-27)
  → 'github:nixos/nix/4acc684ef7b3117c6d6ac12837398a0008a53d85' (2023-02-22)
• Updated input 'hydra/nix/nixpkgs':
    'github:NixOS/nixpkgs/365e1b3a859281cf11b94f87231adeabbdd878a2' (2022-07-13)
  → 'github:NixOS/nixpkgs/04a75b2eecc0acf6239acf9dd04485ff8d14f425' (2022-12-08)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/6c8644fc37b6e141cbfa6c7dc8d98846c4ff0c2e' (2023-01-11)
  → 'github:NixOS/nixpkgs/555daa9d339b3df75e58ee558a4fec98ea92521e' (2023-04-18)

Artturin committed:
changes for update

- users.users.bemeurer.shell is set to fish, but
programs.fish.enable is not true. This will cause the fish
shell to lack the basic nix directories in its PATH and might make
logging in as that user impossible. You can fix it with:
programs.fish.enable = true;

jonringer commented on configuration.nix:
```diff
@@ -254,6 +256,7 @@
   };
 
   # aka. lovesegfault
+  programs.fish.enable = true;
```
do you mind moving this adjacent to the zsh stuff?

Artturin committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'hydra':
    'github:NixOS/hydra/082495e34e094cae1eb49dbfc5648938e23c6355' (2023-03-27)
  → 'github:NixOS/hydra/13ef4e3c5d87bc6f68c91a36d78cdc7d589d8ff2' (2023-05-08)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/555daa9d339b3df75e58ee558a4fec98ea92521e' (2023-04-18)
  → 'github:NixOS/nixpkgs/963006aab35e3e8ebbf6052b6bf4ea712fdd3c28' (2023-05-16)
